en/sql/query/show.rst

```diff
@@ -1837,3 +1837,67 @@ Num_busy_workers            INT             The number of busy worker threads of
 Num_connection_workers      INT             The number of connection worker threads of the queue
 =========================== =============== =======================================================
 
+SHOW PAGE BUFFER STATUS
+-----------------------
+
+It shows the status of data page buffer pool.
```

Comment:
Why do you use "data"? it is only for data pages? It can confuse with the "data page"

Reply:
@wrlawodms I referred to the name of the "data_buffer_size" or "data_buffer_pages" system property, and I think the term "data buffer cache" or "data buffer" is usually used. However, as in your opinion, I agree that the use of "data page buffer" can cause misunderstanding and confusion. So how about changing "data page buffer pool" to "data buffer pool"? Or is your suggestion to use "page buffer pool"?

Reply:
I thought it looks confused because of this metric, but "data page buffer pool" makes sense to discriminate with log page buffer.
> Num_data_pages :            The number of data pages in the buffer pool